src/utils/browser.js

```diff
@@ -49,3 +51,19 @@ export const getCookie = name => {
   }
   return decodeURI(dc.substring(begin + prefix.length, end));
 };
+
+export const buildDynamicURL = (url, qsParams, dynamicQsValues = []) => {
```

Comment:
Do you send `dynamicQsValues` as an object but set the default value as an array?